ui/app/src/utils/path.ts

```diff
@@ -57,29 +56,6 @@ export function parseQueryString(): ParsedQuery {
   return parse(window.location.search);
 }
 
-// TODO: an initial path with trailing `/` breaks
-export function itemsFromPath(path: string, root: string, items: LookupTable<DetailedItem>): DetailedItem[] {
-  const rootWithIndex = withIndex(root);
```

Comment:
Where did this go?

Reply:
This is not needed now, because we are using getIndividualPaths(), and only saving the path/url on the state